src/browser-lib/avo/Avo.ts

```diff
@@ -1141,6 +1141,7 @@ export const PageName = {
   'CURRICULUM_OVERVIEW': 'Curriculum Overview',
   'CURRICULUM_UNIT_SEQUENCE': 'Curriculum Unit Sequence',
   'CURRICULUM_DOWNLOADS': 'Curriculum Downloads',
+  'EARLY_RELEASE_UNITS': 'Early Release Units'
```

Comment:
Did you add this or was it automatically added from avo?

Reply:
I added this as in `url.ts` it required a value for `AnalyticsPageName` value - I have no idea how these values are used though or if there is a knock on effect in the analytics to be mindful of?

Reply:
Avo.ts is generated automatically so we shouldn't edit it directly. I suggest removing this for now and writing a ticket to add  EARLY_RELEASE_UNITS to AnalyticsPageName in Avo. Maybe check in Verity but I don't think it's a priority to add tracking to the early release page now.  Happy to pair and go through this in more detail if it is. Here are the docs.

https://www.notion.so/oaknationalacademy/Avo-5ff21cbd33994c84907c83c6af5a25ee

Reply:
@JBR90 Aha I see thanks! I think since [Ben's PR](https://github.com/oaknational/Oak-Web-Application/pull/1909/files#diff-89a5d358f9c3d0fd56861e3060f31d592b8ad78a406f5b393f8eb861f5c47c62) this change is already in main now so is no longer showing in the latest diff - presumably no new ticket will be required?